packages/webview_flutter/lib/webview_flutter.dart

```diff
@@ -469,6 +469,32 @@ class WebViewController {
   }
 }
 
+/// Manages cookies pertaining to all [WebView]s.
+class CookieManager {
+  factory CookieManager() {
+    if (_instance == null) {
+      final MethodChannel methodChannel =
+          const MethodChannel('plugins.flutter.io/cookie_manager');
+      _instance = CookieManager.private(methodChannel);
+    }
+    return _instance;
+  }
+
+  CookieManager.private(this._channel);
+
+  static CookieManager _instance;
+  final MethodChannel _channel;
+
+  /// Clears all cookies.
+  ///
+  /// This is supported for >= IOS 9 and Android api level >= 16.
```

Comment:
no need to mention Android API level (16 is the minimum level supported by Flutter anyway)